x-pack/plugins/lens/public/state_management/types.ts

```diff
@@ -38,7 +38,6 @@ export interface PreviewState {
   visualization: VisualizationState;
   datasourceStates: DatasourceStates;
   activeData?: TableInspectorAdapter;
-  requestWarnings?: string[];
```

Comment:
use of these was removed in https://github.com/elastic/kibana/pull/147818, but this was left over in the types